git_machete/docs.py

```diff
@@ -303,7 +303,7 @@
     """,
     "github": """
         <b>Usage: git machete github <subcommand></b>
-        where <subcommand> is one of: `anno-prs`, `checkout-prs`, `create-pr`, `retarget-pr`, `fetch-prs`.
+        where <subcommand> is one of: `anno-prs`, `checkout-prs`, `create-pr`, `fetch-prs`, `retarget-pr`.
```

Comment:
Okay, so a high-level consideration about naming of subcommands...

![goncern](https://user-images.githubusercontent.com/3383210/152140433-b3129da8-792e-4eeb-8350-1a79d96648ad.png)

Reply:
Let's consider what the word _checkout a branch_ is supposed to mean in the context of Github PRs:

1. if it means _create a local branch out of the remote branch which is the head of a PR + add it to .git/machete_, then both commands do that exact thing:
   `checkout-prs` should stay `checkout-prs`,
   `fetch-prs` shouldn't really be a separate subcommand &mdash; it should be merged into `checkout-prs`

Reply:
But!

2. if it means _make HEAD point to this branch_ (as is usually does), then:
   * `checkout-prs` should indeed be renamed to singular `checkout-pr` (as you've suggested previously &mdash; my apologies). Since semantic versioning would require making a 4.0.0 release for removing a subcommand... we can instead just mark `checkout-prs` as deprecated and release as 3.8.0. See [release v3.0.0](https://github.com/VirtusLab/git-machete/releases/tag/v3.0.0) &mdash; it was dedicated to removing such deprecated misnomers, like `prune-branches` renamed to `delete-unmanaged`, or `infer` renamed to `discover` soon after it's been first added.
   * `fetch-prs` is acceptable... although I'm a bit unconvinced about the `fetch` verb, since it already has a meaning in gitspeak (well, `git fetch`), and this subcommand is doing much more than just fetching branches :/

Reply:
WDYT? 🤔

Reply:
@PawelLipski I would go for option 2, as for the name,i chose `fetch-prs` because its intuitive and already has a meaning in git (fetching branches, tags, etc) so it should be easy to figure out the `fetch-prs` meaning.

Reply:
Okay! let's go for it then